near-std/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "near-std"
```

Comment:
I would like to discuss:

Do you believe it makes sense to have a separate crate for this?

Do you think it makes sense to have all the standards in a single crate?

FYI, my initial thought was that this is NEAR implementation of Rust std, so I would prefer to have another name, which will also reflect the fact that it is a crate for contracts (currently I would consider it to be a part of nearcore   rather than near-sdk)

Reply:
Yes, I think separate crate make sense, containing all official compositional standards - standards that will be extended by other logic and composed (e.g. tokens, upgradability, etc). Some utils can go into sdk crate proper, but it then will end with overloading SDK with tons of utils. I think release schedule can be pretty different between these two crates.

This is a crate I've been using between all my contracts and actually other folks are starting to use them as well: https://github.com/search?l=TOML&q=near-lib&type=Code

My initial name was near-lib. We can call it near-sdk-lib?